### ISSUE TITLE
api tasks extra metadata attribute

### ISSUE THREAD

**Issue by MauHernandez** [learningequality/kolibri#2676]
### Observed behaviour
@rtibbles when trying to test #2275 I found this. 

When trying to import a channel it showed some errors showed below
![screen shot 2017-11-22 at 7 54 18 pm](https://user-images.githubusercontent.com/13241119/33155516-2805b19e-cfbf-11e7-93bf-e66b752d4d16.png)

![screen shot 2017-11-22 at 7 54 40 pm](https://user-images.githubusercontent.com/13241119/33155525-2dbb51a2-cfbf-11e7-8745-90f8e71f06fc.png)

### User-facing consequences

It was not possible to import new channel

### Errors and logs

Relevant errors and tracebacks from:

[log.txt](https://github.com/learningequality/kolibri/files/1497818/log.txt)

### Steps to reproduce

1. Create new branch directly out of current version of develop
2. Access `http://127.0.0.1:8000/management/device/#/content`

### Context

* Kolibri version: 0.6.1 (I believe)
* Operating system: Mac OS High Sierra 10.13.1
* Browser: Chrome Version 62.0.3202.94 (Official Build) (64-bit)

**Comment by rtibbles** [comment]
Hi @MauHernandez this is due to an out of date version of icequebe. Please reinstall your Python dependencies!

**Comment by MauHernandez** [comment]
hey @rtibbles I just performed:

```# Python requirements
(kolibri)$ pip install -r requirements.txt
(kolibri)$ pip install -r requirements/dev.txt
(kolibri)$ pip install -e .
(kolibri)$ yarn install
```

and the problem persists.

**Comment by lyw07** [comment]
I had this problem before as well, and I think it was gone after I switched to a new virtual environment. Are you still using the old virtual environment that was created before you created the new branch? Do you mind trying with setting up a new virtual environment?

**Comment by lyw07** [comment]
Sorry I think what I said above won't fix the error, since the error is caused by some jobs not cleaned up in the `tasks/api/`. So can you try with deleting `~/.kolibri/job_storage.sqlite3`? Thanks!

**Comment by indirectlylit** [comment]
should be fixed by #2734
